### PR TITLE
Why not Essential?

### PR DESCRIPTION
**Pull request by KTrain5169**
branch is also slightly outdated I think

### CODE REVIEW

File: pages/info.vue [hunk 1]
```diff
@@ -1,35 +1,74 @@
...
+            <span class="text-[1.5rem] sm:text-[2rem] md:text-[3rem] lg:text-[4rem] internalFont flex flex-row flex-wrap gap-3">
+                <span class="font-bold"> WHY SHOULD YOU NOT USE <span
+                        class="brand-gradient">ESSENTIAL?</span></span>
+            </span>
 
-                                                                                        </ProjectTitleBar>
+        </ProjectTitleBar>
 
-                                                                                                <ProjectParagraph>
-                                                                                                            Don't be forced to use Essential. There are countless open-source alternatives to help you.
```
Comment (KTrain5169) [inline pages/info.vue:15]: wtf what happened to the indenting here

File: pages/info.vue [hunk 2]
```diff
@@ -1,35 +1,74 @@
 <template>
 
-        <projectContainer class="flex gap-8">
```
Comment (KTrain5169) [inline pages/info.vue:3]: the indenting here is jank

File: pages/info.vue [hunk 3]
```diff
@@ -1,35 +1,74 @@
...
+            </span>
 
-                                                                                        </ProjectTitleBar>
+        </ProjectTitleBar>
 
-                                                                                                <ProjectParagraph>
-                                                                                                            Don't be forced to use Essential. There are countless open-source alternatives to help you.
-                                                                                                                    </ProjectParagraph>
+        <ProjectParagraph>
+            Why are we recommending not to use Essential? This page will list our reasons why you should use alternatives.
```
Comment (KTrain5169) [inline pages/info.vue:15]: might need better wording

File: pages/info.vue [hunk 4]
```diff
@@ -1,35 +1,74 @@
...
+
+        <projectParagraph>
+
+            Minecraft modding can be very limitless, but this is because of one thing: Minecraft mods have access to (almost) your entire computer. This makes it very risky to run mods, which means that we have to trust the code that we are executing.
+
+        </projectParagraph>
+
+        <projectParagraph>
+            
+            But, the very nature of closed-source and obfuscation hinders that in a very big way. If we cannot see the code that is being ran, how can we trust it won't be looking through our files? This is why mods being open-source is a big deal.
```
Comment (KTrain5169) [inline pages/info.vue:50]: and unobfuscated?

**Review by KTrain5169 — COMMENTED** [review]
interesting indenting currently

**Comment by Gaming32** [comment]
I will note that Essential is *not* obfuscated in the slightest.

**Comment by KTrain5169** [comment]
I swear it was? If so then I’ll adjust it next time I can get to my computer and start working on this PR. Thanks for letting me know!

**Review by Nitrrine — CHANGES_REQUESTED** [review]
I think it will look better if you make buttons on index page to be in row instead of in column.

**Comment by KTrain5169** [comment]
image
lmao

**Comment by KTrain5169** [comment]
> image
> 
> It looks good! But make sure so it will transform to column from row on mobile

goddamn it I need to figure out how

**Comment by KTrain5169** [comment]
I DID IT YAYY

**Review by Nitrrine — APPROVED** [review]
LGTM!

**Review by blryface — CHANGES_REQUESTED** [review]
Screenshot_20240705-101046
Text should be smaller on mobile, and we need to make the info sections shorter so people will actually read it, giving more information in a more general section at the end or something

**Comment by KTrain5169** [comment]
> Screenshot_20240705-101046 Text should be smaller on mobile, and we need to make the info sections shorter so people will actually read it, giving more information in a more general section at the end or something

I think it would be better to put a short section on index then, after that we can have a learn more button instead of a why not

**Comment by blryface** [comment]
Sounds good

**Comment by blryface** [comment]
Will your branch being *very* outdated not cause issues?

**Review by blryface — APPROVED** [review]

File: pages/index.vue [hunk 5]
```diff
@@ -1,32 +1,43 @@
...
 </template>
 <!-- ⬇️ is supposed to -->
 <script setup lang="ts">
+import ProjectTitleBar from '../components/projectTitleBar.vue';
+
 useSeoMeta({
-    title: 'Home | Not Essential',
-    description: "Essential has many issues. This website tells why, and provides alternatives.",
-    ogImage: "/img/icon512.png",
+  title: "Home | Not Essential",
```
Comment (blryface) [inline pages/index.vue:37]: With the other changes that happened I have a feeling this PR will cause uh issues

File: pages/info.vue
Comment (blryface) [inline pages/info.vue:1]: This one lgtm

**Review by blryface — CHANGES_REQUESTED** [review]
Overall looks good, but try updating your branch to prevent issues with new features

**Comment by KTrain5169** [comment]
Closing PR, this branch is way too big and messy LMAO

Development will now be moved to the 12-add-the-info branch on blryface/notessential

**Pull request closed without merging**